pyartcd/pyartcd/pipelines/build_conforma_verify.py

```
@@ -185,13 +260,85 @@ async def _find_latest_fbcs(self) -> list[KonfluxRecord]:
                 records_by_name[record.name] = record
         return list(records_by_name.values())
 
-    async def _verify_records(self, records: list[KonfluxRecord], build_type: str = "image") -> bool:
-        """Run EC verification on a list of build records.
+    async def _find_operator_names(self) -> set[str]:
+        cmd = self._doozer_base_command + ['olm-bundle:list-olm-operators', '--output-format', 'distgit-key']
```

Ruff (0.15.21)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)